.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
```

Comment:
Does this also include sub directories like `packages/sit-onyx`?

Reply:
I didn't find any definitiv answer about this. Dependabot supports pnpm and the `pnpm-lock.yaml` file, so I think so. We will need to test it to find out.